### PR TITLE
Some easy fixes for `mrgn-docs`

### PR DESCRIPTION
**Pull request by willgraf**
Update consumers docs to reflect the submodule approach currently in master.

Ignore docs folder in `pytest.ini` to pass the travis tests.

Set up (and comment out) some travis tests for the docs. It has a python2 sphinx incompatibility and some issues importing the gensidebar. We can resolve that later.

Added new lines to the end of all files.

### CODE REVIEW

File: docs/source/redis_consumer.consumers.rst [hunk 1]
```diff
@@ -1,7 +1,42 @@
-redis_consumer.consumers module
-===============================
+deepcell.datasets package
```
Comment (msschwartz21) [inline docs/source/redis_consumer.consumers.rst]: Change name back to redis_consumer.consumers module

File: docs/source/redis_consumer.consumers.rst [hunk 2]
```diff
@@ -1,7 +1,42 @@
...
+=========================
+
+Submodules
+----------
+
+redis_consumer.consumers.base_consumer module
+----------------------------------
```
Comment (msschwartz21) [inline docs/source/redis_consumer.consumers.rst]: Header underlines need to be at least as long as the title

File: docs/source/redis_consumer.consumers.rst (hunk 1 above)
Comment (willgraf) [reply docs/source/redis_consumer.consumers.rst]: Should it be module or package?

I copied from deepcell.datasets, which had package.

**Comment by willgraf** [comment]
> Just to confirm, did this PR include deleting the processing files and moving to dependence on deepcell-toolbox?

This PR does include an update from the master branch. `processing.py` still exists, but now just imports functions from `deepcell_toolbox`.

**Pull request merged**